public/locales/en/translation.json

```diff
@@ -38,11 +38,12 @@
             "validated": "This scenario is validated, you cannot edit its parameters",
             "noTabs": "No parameters to edit"
           },
-          "discard": "Discard modifications",
+          "discard": "DISCARD",
           "edit": "Edit parameters",
-          "launch": "Launch scenario",
+          "launch": "LAUNCH",
           "update": {
-            "launch": "Update and launch scenario"
+            "update": "SAVE",
```

Comment:
`scenario.parameters.update.update` is redundant, maybe we could move this label up to `scenario.parameters.save` ? This way, it would be at the same level as the other actions labels (discard, launch)